zerver/openapi/zulip.yaml

```diff
@@ -9096,6 +9096,49 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /users/me/test_notify:
+    post:
+      operationId: test-notify
+      summary: Sends a test notifications to the user's mobile device(s)
+      tags: ["users"]
```

Comment:
not sure about this tag, but matches better than any other tag we currently use i suppose

Reply:
I think this should probably be in a new "mobile" tag.

Reply:
Fixed